web/src/App.svelte

```diff
@@ -157,10 +166,17 @@
       class="pico"
       style="display: flex; align-items: center; gap: 8px;"
     >
-      <button class="outline" on:click={() => ($showAbout = true)}>
+      <button class="outline" on:click={() => (showAbout = true)}>
         <img src={logo} style="height: 32px;" alt="A/B Street logo" />
       </button>
       <span bind:this={topDiv} style="width: 100%" />
+      <button
```

Comment:
It seems useful to always have a link to the manual handy. Hopefully the help icon is obvious enough. I also tried this one:
![image](https://github.com/user-attachments/assets/bb7ae21d-8235-4739-b5da-dd79de179721)